docs/dev/fundamentals/zkSync.md

```diff
@@ -0,0 +1,72 @@
+
+zkSync 2.0 is made to look and feel like Ethereum, but with lower fees. Just like on Ethereum, smart contracts are written in Solidity/Vyper and can be called using the same clients as the other EVM-compatible chains.
+
+zkSync 2.0 doesn't require you to register a separate private key to use before usage, it supports existing Ethereum wallets which work out of the box.
+
+
+At this time zkSync is solely run and operated by the zkSync team's servers, and is therefore centralized; however, this will be be transitioned to a decentralized system in the near future.
+
+## zkSync in comparison
+
+zkSync [stands out remarkably](https://blog.matter-labs.io/evaluating-ethereum-l2-scaling-solutions-a-comparison-framework-b6b2f410f955) in security and usability among existing L2 scaling solutions. 
+Thanks to the combination of cutting-edge cryptography and on-chain data availability, zkRollup (the core network of zkSync) is the only L2 scaling solution that doesn't require any operational activity to keep the funds safe. 
+For example, users can go offline and still be able to withdraw their assets safely when they come back, even if the zkRollup validators are no longer around.
+
+In the following sections we will cover some attributes of zkSync.
+
+## Transaction types
+
+zkSync supports Ethereum's "legacy" (pre-EIP2718) transaction types, EIP1559 transaction type, and its custom EIP712 transactions. You can use transactions of this type to use zkSync-specific features like account abstraction. 
+Additionally, it is only possible to deploy smart contracts with this type of transaction.
+
+Knowing the details about the transaction format is not required to use zkSync's SDK, but if you are curious, you can read more about it [here](../../api/api.md#eip712).
+
+
+## Confirmations and finality
+
+On zkSync, each transaction is at one of the four stages:
+
+- `Pending`: The transaction was received by the operator, but it has not been processed yet.
+- `Processed`: The transaction is processed by the operator and is confirmed to be included in the next block.
+- `Committed`: The transaction state diffs were published on Ethereum.
+- `Finalized`: The SNARK validity proof for the transaction has been submitted and verified by the smart contract. After this step, the transaction is considered to be final.
+
+The typical time for a transaction to go from `Processed` to `Finalized` is a couple of hours at the current stage.
+
+Please note that for developer convenience, we usually treat the `Processed` and `Committed` states as a single stage called `Committed` since they have no difference from the UX/DexEx standpoints.
+
+
+## zkSync features
+
+- ETH and ERC20 token transfers with instant confirmations and 10-minute finality on L1
+- Transaction fees are extremely low (1/100th of the mainnet cost for ERC20 tokens and 1/30th for ETH transfers).
+- No registration is required to receive funds.
```

Comment:
This was taken from the documentation of zkSync 1.x :) 

This point was emphasized in contract to some other zkRollups that required registration both to receive funds and send funds. On zkSync the users could receive funds without registration, but they still had to do a small registration transaction before they could actually do anything with these funds. So no registration to receive, but there is a registration to transact.

On zkSync 2.0 it is the same as on Ethereum or Optimistic rollups: no need to register at all. So probably it is better to remove this point altogether